### DISCUSSION TITLE
Welcome to metric-learn Discussions!

### DISCUSSION THREAD

**Discussion by wdevazelhes** [scikit-learn-contrib/metric-learn#315] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wdevazelhes** [comment]
Hey, I just saw the discussion feature on github, I thought we could give it a try. Above is the automatic welcome message generated by github to get started with discussion.

**Comment by terrytangyuan** [comment]
This is definitely great to have! Though we should probably add a issue template to differentiate bug reports and general discussions. An example here: https://github.com/argoproj/argo-workflows/blob/master/.github/ISSUE_TEMPLATE/config.yml

**Reply by wdevazelhes** [reply]
@terrytangyuan I agree, thanks for the link!
I opened a PR for that #316
